.github/workflows/pr-test-lint.yaml

```diff
@@ -66,3 +66,35 @@ jobs:
         env:
           API_VERSION: x5.0
           CNSPEC_VERSION: ${{ env.CNSPEC_VERSION }}
+
+  debug:
```

Comment:
We can remove this later, but currently I might need some data from the dump, to check the workflow in case of an error.

Reply:
i'm scared of us forgetting it and inadvertently dumping some secret in the future - can we comment this out for now and only add if if we really need it?

Reply:
https://github.com/mondoohq/packer-plugin-cnspec/pull/339